src/convertToObject.js

```diff
@@ -3,16 +3,38 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const cleanedUpString = sourceString
+    .split('\n')
+    .join('')
+    .trim()
+    .split(';')
+    .map(el => el.split(':'));
+
+  const result = {};
+
+  cleanedUpString.forEach(element => {
+    if (element.length > 1) {
```

Comment:
```suggestion
   .filter(element=> element.includes(':'))
   and then =>
   .map(el => el.split(':'));
```
You could implement a filter iteration with a rule that will allow us to work only with elements that have ':' 
So basically you could implement this filter from the beginning and then you wouldnt need to worry about aplying the if statement for each element